Pod/Classes/PINRemoteImageManager.m

```diff
@@ -751,6 +751,25 @@ - (void)downloadImageWithURL:(NSURL *)url
     [self unlock];
 }
 
+-(BOOL) insertImageDataIntoCache:(nonnull NSData*)data withURL:(nonnull NSURL *)url processorKey:(nullable NSString *)processorKey {
+  
+  if (url != nil) {
+    NSString *key = [self cacheKeyForURL:url processorKey:processorKey];
+    
+    PINRemoteImageManagerDownloadOptions options = PINRemoteImageManagerDownloadOptionsSkipDecode & PINRemoteImageManagerDownloadOptionsSkipEarlyCheck;
+    PINRemoteImageMemoryContainer *container = [[PINRemoteImageMemoryContainer alloc] init];
+    container.data = data;
+    
+    return [self materializeAndCacheObject: container cacheInDisk: data additionalCost:0 key:key options:options outImage: nil outAltRep: nil];
```

Comment:
I think additionalCost is used when dealing with processed images. If this method is going to take a URL and a processorKey, it should also probably take a processingCost parameter and pass that in to the additional cost?